image/system_a/data/lang/English.json

```diff
@@ -715,6 +715,7 @@
   "app_bell_relaxation_looped": "looped",
   "app_bell_relaxation_loop_description": "the song will play until you turn it off",
   "app_bell_relaxation_error_message": "Unsupported media type",
+  "app_bell_relaxation_limit_error_message": "<text>File limit exceeded.<br />Not all files may display<br />correctly</text>",
```

Comment:
```suggestion
  "app_bell_relaxation_limit_error_message": "<text>File limit exceeded.<br />Not all files may be displayed<br />correctly</text>",
```

Reply:
This text copied from the design, and for some online tools it looks grammatically correct

Reply:
Done.